docs/workflows/webauthn.md

```diff
@@ -1,44 +1,36 @@
-# Sign up and sign in users with Webauthn using strong authenticator
+# Security Keys with WebAuthn
 
-User can sign up with webauthn only if email verification is disabled. When email verification is enabled, user must verify it's email, login via password, magic link or ther credentials and then add webauthn authenticator via `/user/webauthn/add` endpoint instead. After that user can sign in using the webauthn authenticator.
+Hasura-auth implements the WebAuthn protocol to sign in with security keys.
 
-## Sign up
+A user needs first to sign up with another method, for instance email+password, passwordless email or Oauth, then to add their security key to their account.
+
+Once the security key is added, it is then possible to sign in with it, using the email as a user name.
+
+## Add a security key
+
+Users can add multiple authenticators, for example when they need to login from multiple devices or browsers. To do that, they should have a valid session.
```

Comment:
Is "authenticators" the same thing as "security keys"? 

Asking because we're using the terminology "security keys" in the header for this secion.

Reply:
What do you mean by "valid session"? Do you mean users must be signed in when adding a security key?

Reply:
yes - catching up previous explanations